src/docs/development/add-to-app/android/project-setup.md

```diff
@@ -4,4 +4,226 @@ short-title: Integrate Flutter
 description: Learn how to integrate a Flutter module into your existing Android project.
 ---
 
-TODO(blasten): document.
\ No newline at end of file
+Flutter can be embedded into your existing Android application in a piecemeal
+fashion as a source code sub-project or as AARs.
+
+The integration flow can be done via the Android Studio IDE with the
+[Flutter plugin](https://plugins.jetbrains.com/plugin/9212-flutter) or manually.
+
+{{site.alert.warning}}
+Your existing Android app may currently support architectures such as `mips`
+or `x86`/`x86_64`. Flutter currently [only supports](/docs/resources/faq#what-devices-and-os-versions-does-flutter-run-on)
+building AOT libraries for `armeabi-v7a` and `arm64-v8a`.
+
+Consider using the [`abiFilters`](http://google.github.io/android-gradle-dsl/current/com.android.build.gradle.internal.dsl.NdkOptions.html)
```

Comment:
This needs an example. Some folks run into issues when they use the Crashlytic plugin.

Reply:
I added something but you might have to look it over again. I'm not sure if it's going to mess with emulators on debug mode.